testing/microlith-test/integration/prometheus_alert_overrides/prometheus_alert_overrides.bats

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bats
+
+# Copyright 2021 Couchbase, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file  except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the  License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load "$BATS_SUPPORT_ROOT/load.bash"
+load "$BATS_ASSERT_ROOT/load.bash"
+load "$BATS_FILE_ROOT/load.bash"
+load "../helpers"
+
+export CMOS_IMAGE=${CMOS_IMAGE:-couchbase/observability-stack:v1}
+
+setup() {
+    if [ "$TEST_INTEGRATION" == "true" ]; then
+        skip "Skipping integration tests"
+    fi
+    find couchbase custom overrides -regex '.*\.ya?ml\.orig$' -delete
+}
+
+teardown() {
+    if [ "$SKIP_TEARDOWN" == "true" ]; then
+        echo "# Skipping teardown. Make sure to manually run the commands in teardown()." >&3
+        return
+    fi
+    docker-compose rm -v --force --stop
```

Comment:
We don't want to fail if set up fails so maybe hide with a `run` call

Reply:
This is teardown, not setup, should be fine no?

Reply:
It always runs even if setup fails so I don't want a load of spam saying failed to teardown stuff that also failed to start up

Reply:
Ah okay, I'll change that